.travis.yml

```diff
@@ -5,14 +5,14 @@ node_js:
 - '6.15.1'      #boron
 - '8.14.0'      #carbon
 - '10.14.2'     #dubnium
-- stable
+- '14.13.1'     #stable
 
```

Comment:
This will be changed back to 'stable' version 15.0.1 once the tests that were failing with that version are fixed.